### ISSUE TITLE
[Plex] Error getting state (status code 401): null

### ISSUE THREAD

**Issue by nebtechmsp** [mpbzh/homebridge-plex#17]
Hi there,
I'm trying to get this set up but I keep getting the error "[Plex] Error getting state (status code 401): null"

I have verified the token is correct and renewed it. I have also debugged the token and it is giving me the correct media container response. I have tried manually specifying the server IP (even though it is on the localhost) and I have also tried deleting my config.json and recreating it using nano and li in an attempt to prevent formatting issues, but the issue remains.

Please help!

**Comment by lukezona** [comment]
Hi, I'm having the same issue. 

Have you solved it?

**Comment by nebtechmsp** [comment]
I resolved this issue by removing "X-Plex-Token=" from my token (it wasn't clear it needed to be the token without this prefix). I found even after fixing this though that the plugin still wasn't working and responding when I played something through Plex (it couldn't see my Chromecast). So in the end, I found another homebridge plugin that allowed me to monitor when my Chromecast was in use and use that as a trigger. Worked like a charm first try!
